tests/page-objects/default/reports/reports.wdio.page.js

```diff
@@ -205,51 +220,44 @@ const isReportSelected = async (uuid) => {
   return await (await checkbox).isExisting();
 };
 
-const selectReports = async (uuids) => {
+const toggleSelectReport = async (selectReport, uuids, verifyFn, shouldHideElements = false) => {
   for (const uuid of uuids) {
-    if (!(await isReportSelected(uuid))) {
-      await (await reportCheckbox(uuid)).click();
+    const isSelected = await isReportSelected(uuid);
+    if ((selectReport && !isSelected) || (!selectReport && isSelected)) {
+      await (await leftPanelSelectors.reportCheckbox(uuid)).click();
     }
   }
-  return verifyMultiselectElementsDisplay();
+  return verifyFn(shouldHideElements);
```

Comment:
```suggestion
  return verifyMultiselectElementsDisplay(shouldHideElements);
```

Reply:
I pass the function as parameter because I am using the same function in the reports page for `default-mobile`, [here](https://github.com/medic/cht-core/blob/8704-clean-up-and-organize-reports-folder/tests/page-objects/default-mobile/reports/reports.wdio.page.js#L36-L44), and the method `verifyMultiselectElementsDisplay` is different for `default` and `default-mobile`